sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-2021-2022-awards-amount-BCAG.e2e-spec.ts

```diff
@@ -127,4 +127,31 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-BC
     expect(calculatedAssessment.variables.awardEligibilityBCAG).toBe(false);
     expect(calculatedAssessment.variables.provincialAwardNetBCAGAmount).toBe(0);
   });
+
+  it("Should determine provincialAwardNetBCAGAmount when awardEligibilityBCAG is true and provincialAwardPartTimeBCAGAmount is null", async () => {
```

Comment:
I would recommend the description to be more user friendly like `no BCAG awarded in the program year previously`